### PR TITLE
ci: fix build runner, ensure jest exits when finished

### PR DESCRIPTION
**Pull request by jakebolam**
**What**:
Fix the build so we can release. Jest was hanging open in some cases

**Why**:
Build is broken

**How**:
Me

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [x] Ready to be merged 
- [x] Added myself to contributors table

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "yargs": "^10.0.3"
   },
   "devDependencies": {
-    "kcd-scripts": "^0.30.0",
+    "kcd-scripts": "^0.30.4",
```
Comment (jakebolam) [inline package.json]: not required, but good to do anyway

File: jest.config.js [hunk 2]
```diff
@@ -9,4 +9,5 @@ module.exports = Object.assign(jestConfig, {
       statements: 50,
     },
   },
+  forceExit: true
```
Comment (kentcdodds) [inline jest.config.js]: Interesting 🤔

**Comment by jakebolam** [comment]
@kentcdodds jest has finished running, but there are still open async handles (maybe something in nock, I couldn't find the specific handles) jest is waiting for these to finish but they never do. Usually this is things like DB connection etc

**Comment by kentcdodds** [comment]
We should probably find out what they are and close the connections manually

**Pull request merged**